cypress/support/step_definitions/fireboltCalls.js

```diff
@@ -365,3 +365,115 @@ Given('device is rebooted', () => {
+    KeyPressSequence = Cypress.env(CONSTANTS.RUNTIME).intent.keyPressSequence;
+  } else if (
+    // Check if defaultKeyPressSequence is defined for the specific appId in app_metadata
+    Cypress.env('app_metadata') &&
+    Cypress.env('app_metadata')[appId] &&
+    Cypress.env('app_metadata')[appId].defaultKeyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env('app_metadata')[appId].defaultKeyPressSequence;
+  } else if (
+    // Check if defaultKeyPressSequence is defined in the app_metadata globally
+    Cypress.env('app_metadata') &&
+    Cypress.env('app_metadata').defaultKeyPressSequence
+  ) {
+    KeyPressSequence = Cypress.env('app_metadata').defaultKeyPressSequence;
+  } else {
+    // If no keyPressSequence is found, throw an error with details from the app_metadata file
+    const appMetadataJSON = require('../../fixtures/objects/appData/app_metadata.json');
+    throw new Error(
+      `Expected KeyPressSequence was not found for ${appId} in app_metadata.json. More details on app_metadata present in: ${appMetadataJSON}`
+    );
+  }
+  cy.exitAppSession('dismissApp', KeyPressSequence.dismiss).then((response) => {
+    fireLog.info(`Response from platform: ${JSON.stringify(response)}`);
+  });
+});
+
+/**
+ * @module fireboltCalls
+ * @function And 3rd party '(.+)' app should be exited
+ * @description To validate that the app is dismissed
+ * @param {String} app - app name.
+ * @example
+ * Then 3rd party 'firebolt' app should be exited
+ */
+Given(/3rd party '(.+)' app should be exited$/, async (app) => {
+  // getAppState validation
+  const appId = Cypress.env(CONSTANTS.CURRENT_APP_ID);
+  const requestMapForGetAppState = {
+    method: CONSTANTS.REQUEST_OVERRIDE_CALLS.GETAPPSTATE,
+    params: appId,
+  };
+  fireLog.info(
+    `Sending request to fetch ${appId} app state: ${JSON.stringify(requestMapForGetAppState)}`
+  );
+  cy.sendMessagetoPlatforms(requestMapForGetAppState)
+    .then((response) => {
+      const responseString = JSON.stringify(response);
+      if (
+        // Check if response exists and contains appState, visibilityState
+        // and if appState is INACTIVE and visibilityState is VISIBLE
+        response &&
+        response.appState &&
+        response.visibilityState &&
+        response.appState.toUpperCase() === CONSTANTS.LIFECYCLE_STATES.INACTIVE &&
+        response.visibilityState.toUpperCase() === CONSTANTS.VISIBLE
+      ) {
+        fireLog.info(
+          `State validation successful: Current state of ${appId} app is ${responseString} as expected`
+        );
+      } else if (
+        // Check if appState is missing, visibilityState exists, and visibilityState is VISIBLE
+        response &&
+        !response.appState &&
+        response.visibilityState &&
+        response.visibilityState.toUpperCase() === CONSTANTS.VISIBLE
+      ) {
+        fireLog.info(
+          `State validation successful: Current state of ${appId} app is ${responseString} as expected`
+        );
```

Comment:
line 453 is not correct. we are not getting any app state and we are not validating app state here

Reply:
updated the logs